packages/nocodb/src/version-upgrader/upgraders/0227002_ncBrokenLinkRecovery.ts

```
@@ -0,0 +1,288 @@
+            ncMeta.knex(MetaTable.COL_RELATIONS).insert({
+              ...commonProps,
+              type: RelationTypes.BELONGS_TO,
+              fk_child_column_id: link.fk_child_column_id,
+              fk_parent_column_id: link.fk_parent_column_id,
+            });
+            break;
+          case RelationTypes.ONE_TO_ONE:
+            // todo:
+            const meta = {
+              bt: false,
+            };
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)